### PR TITLE
Create symbol files on Windows release builds

### PR DESCRIPTION
**Pull request by kencooke**
Added --release to windeployqt.exe, otherwise it gets confused and copies debug DLLs instead of release.

It would be best to create the Debug project without this flag.. does anyone know how to implement this in the CMake setup?

### CODE REVIEW

**Comment by hifi-gustavo** [comment]
Build console output for commit: 4dc2c9a58f26cbd8a1dc183215ec69da05c43897 available at: https://gist.github.com/cb6cc3eddc86a39993fa<br/><br/>Installers available for download at: <br/>Windows: http://s3.amazonaws.com/hifi-public/pr-builds/dev-interface-5945.exe<br/>OS X: http://s3.amazonaws.com/hifi-public/pr-builds/dev-interface-5945.dmg

File: cmake/macros/CopyDllsBesideWindowsExecutable.cmake [hunk 1]
```diff
@@ -37,7 +37,7 @@ macro(COPY_DLLS_BESIDE_WINDOWS_EXECUTABLE)
     add_custom_command(
       TARGET ${TARGET_NAME}
       POST_BUILD
-      COMMAND CMD /C "SET PATH=%PATH%;${QT_DIR}/bin && ${WINDEPLOYQT_COMMAND} $<TARGET_FILE:${TARGET_NAME}>"
+      COMMAND CMD /C "SET PATH=%PATH%;${QT_DIR}/bin && ${WINDEPLOYQT_COMMAND} --release $<TARGET_FILE:${TARGET_NAME}>"
```
Comment (jherico) [inline cmake/macros/CopyDllsBesideWindowsExecutable.cmake]: wouldn't this break actual debug builds?

**Comment by kencooke** [comment]
I added some cmake-generator-expression logic to use the correct flag for each build type.  This fixes the existing problem with RelWithDebInfo and avoids creating a new problem with Debug build.

**Comment by hifi-gustavo** [comment]
Build console output for commit: de8848f7bcf693d8e3047e973aac3043b32012c6 available at: https://gist.github.com/cb6cc3eddc86a39993fa<br/><br/>Installers available for download at: <br/>Windows: http://s3.amazonaws.com/hifi-public/pr-builds/dev-interface-5945.exe<br/>OS X: http://s3.amazonaws.com/hifi-public/pr-builds/dev-interface-5945.dmg

**Pull request merged**